workspaces/adventure-pack/goodies/kotlin/src/common/TreeNode.kt

```diff
@@ -0,0 +1,3 @@
+package common
```

Comment:
I was hoping this would work, but unfortunately the checks failed... try running `yarn build-app` in the `workspaces/adventure-pack` directory. Basically the goody extraction code assumes that every directory under `workspaces/adventure-pack/goodies/kotlin/src` will have a file named `Main.kt` and a file named `goody.json`. You can see the implementation for that in [`workspaces/adventure-pack/src/scripts/package-goodies/kotlin/readBaseGoody.ts`](https://github.com/code-chronicles-code/leetcode-curriculum/blob/main/workspaces/adventure-pack/src/scripts/package-goodies/kotlin/readBaseGoody.ts).

What we're doing for now in Java, for example, is copy-pasting the TreeNode class in each goody that needs it. I think we can do that here as well to unblock. If you're up for it, feel free to file an issue about this, or let me know if you'd like me to file it.

Reply:
Also feel free to file issues for anything else that you thought was non-ideal to have to figure out, or that could have been better in the dev experience. I was the only contributor for a while so the repository isn't always friendly to work in, but I'd like to improve on that front.